testbed/tests/trace_test.go

```diff
@@ -59,15 +59,16 @@ func TestTrace10kSPS(t *testing.T) {
 				ExpectedMaxRAM: 85,
 			},
 		},
-		{
-			"AwsXray",
-			testbed.NewOTLPTraceDataSender("localhost", testbed.GetAvailablePort(t)),
-			datareceivers.NewMockAwsXrayDataReceiver(testbed.GetAvailablePort(t)),
-			testbed.ResourceSpec{
-				ExpectedMaxCPU: 75,
-				ExpectedMaxRAM: 85,
-			},
-		},
+		// Disabled due to instability. Issue #1380
+		// {
+		// 	"AwsXray",
+		// 	testbed.NewOTLPTraceDataSender("localhost", testbed.GetAvailablePort(t)),
+		// 	datareceivers.NewMockAwsXrayDataReceiver(testbed.GetAvailablePort(t)),
+		// 	testbed.ResourceSpec{
+		// 		ExpectedMaxCPU: 75,
+		// 		ExpectedMaxRAM: 85,
+		// 	},
+		// },
```

Comment:
Please delete the code, we don't allow commented code. It is already recorded as an issue.